screens/Login.tsx

```
@@ -3,12 +3,11 @@ import { View, StyleSheet, Image, TouchableOpacity } from 'react-native';
 import { ThemedText } from '@/components/ThemedText';
 import { Link, router } from 'expo-router';
 import { LegalDisclaimer } from '@/components/LegalDisclaimer';
+import { Colors } from '@/constants/Colors';
 
-// You'll need to add your app logo image to your assets
-import APP_LOGO from '@/assets/images/react-logo.png';
+const APP_LOGO = require('@/assets/images/react-logo.png');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)